### ISSUE TITLE
Ability to use the pre compiled base from other sources.

### ISSUE THREAD

**Issue by mzs114** [olevole/cbsd#57]
Hi,

For using CBSD on a new machine you need to have internet, else you need to copy `/usr/src` to `/path/src/<freebsd-version>/src`, and compile.

Some users prefer to avoid compiling and use what is built.
Provide a provision like ezjail-admin where you can use the `freebsd-dist` directory from a location, which has doc.txz, kernel.txz, ports.txz, base.txz, games.txz, lib32.txz, src.txz.

With an optical disk it would be like following to create a base jail:

`# mount_cd9660 /dev/cd0 /mnt/`

`# ezjail-admin install -h file:///mnt/usr/freebsd-dist/`

To install the man pages we can use `-m`.

`# ezjail-admin install -m -h file:///mnt/usr/freebsd-dist/`

Now I can use this to create jails.
This will help people who don't have internet or a fast internet link(majority of the world falls into this category).

**Comment by mzs114** [comment]
Ok, after reading the PS to use our own repo in #52, I tried extracting the `base.txz` by creating a new directory  `/path/basejail/base_amd64_amd64_10.2/` on a new installation of CBSD.

`# tar xJf /mnt/usr/freebsd-dist/base.txz  -C /path/basejail/base_amd64_amd64_10.2/`

This worked! 
I could list the base and create new jails:

```
# cbsd basels
NAME  ARCH   TARGET_ARCH  VER
base  amd64  amd64        10.2
```
Please close the ticket if this is ok.

**Comment by olevole** [comment]
Hi! Thanks for feedback. 
This ticket is interesting because it is the second one: https://github.com/olevole/cbsd/issues/52
I probably make possible to obtain a minimum base of working copy (ie, from  /  environment) in the next version of CBSD.

**Comment by mzs114** [comment]
Thanks!
Let us know if you would like some testing.

I am proposing CBSD as a container management system where people are inclining towards Docker/CoreOS/Kubernetes, etc.

In my opinion CBSD does not introduce new layers of management, it makes use of the good old tools(rsync, zfs, etc).

**Comment by olevole** [comment]
*Stage1/2 done* (within https://github.com/olevole/cbsd/commit/79c51fe942d512ce0e01d96eb8ace2caa3c0947c): Implement populate base from current system. No source tree or object file needed. This work only if the version of container == base  system version

*Stage2/2: implement local repo, documentation on site: work in progress.

**Comment by olevole** [comment]
Hello.

**Summarily**: implement in two ways:
1.  Populate base directory from existing master-host environment (only when version of master-host and jail is equal. No internet connection required).
2. Add user's input on get_base stage to to be able to specify path for archive ( somewhere within this commit: https://github.com/olevole/cbsd/commit/3ca1c87585a19bf10fb8f1eafd677802c00883de ).

Will be included in CBSD 10.3.2.
Thanks for contribution's and any ideas/comments.
